### ISSUE TITLE
[BUG] Fatal Error When Using Along Side Elementor Plugin.

### ISSUE THREAD

**Issue by varunsridharan** [wponion/wponion#252]
## Describe the bug
```log
An error of type E_ERROR was caused in line 124 of the file
/********/wp-content/plugins/woocommerce-quick-buy/vendor/wponion/wponion/setup/class-setup.php.
Error message: Uncaught Error: Class
‘WPOnion\Integrations\Page_Builders\Elementor’ not found in
/********/wp-content/plugins/woocommerce-quick-buy/vendor/wponion/wponion/setup/class-setup.php:124
Stack trace:
#0 /********/wp-includes/class-wp-hook.php(287):
WPOnion\Setup::on_wponion_loaded(”)
#1 /********/wp-includes/class-wp-hook.php(311):
WP_Hook->apply_filters(”, Array)
#2 /********/wp-includes/plugin.php(478):
WP_Hook->do_action(Array)
#3 /********/wp-content/plugins/woocommerce-quick-buy/vendor/wponion/wponion/setup/class-setup.php(98):
do_action(‘wponion/loaded’)
#4 /********/wp-content/plugins/woocommerce-quick-buy/vendor/wponion/wponion/setup/class-setup.php(49):
WPOnion\Setup::load_required_files()
#5 /********/wp-content/plugins/woocommerce-quick-buy/vendor/wponio
```

## To Reproduce
1. Install WPOnion
2. Install Elementor
3. Activate Both.

## Screenshots
![image](https://user-images.githubusercontent.com/1884287/84588184-c090a180-ae42-11ea-9373-00d5c65ea00e.png)

**Comment by varunsridharan** [comment]
Fixed in V1.5.2

**Comment by varunsridharan** [comment]
Ref: https://wordpress.org/support/topic/fatal-error-after-updating-16/
